src/components/z-book-cover/styles.css

```diff
@@ -0,0 +1,68 @@
+:host,
+* {
+  box-sizing: border-box;
+}
+
+:host {
+  --z-book-cover-width: 100%;
```

Comment:
Have you considered setting the width to the fallback image’s value? Is there a reason why you preferred 100% ?

Reply:
Probably a leftover of one of the many implementation I changed, maybe `auto` it's a safer value to avoid extra configurations in some edge cases.